docs/multipass.md

```diff
@@ -0,0 +1,52 @@
+# Testing LibreTime with Canonical's Multipass
+
+Ever wanted to test out LibreTime but didn't want to tango with
+Virturalbox, Vagrent, or Docker?
+
+![](https://thumbs.gfycat.com/HauntingDirtyDragon-size_restricted.gif)
+
+Canonical released [Multipass](https://multipass.run), a tool for setting up Ubuntu VMs with cloud-init files in a snap.
+Multipass is available for Windows and macOS, as well as Linux OSes that support snaps.
+
+Similar to Docker, Multipass works through a CLI. To use, clone this repo and then open a Terminal
+(or Command Prompt) inside the `libretime` folder and run
+```
+multipass launch bionic -n ltTEST --cloud-init libretimeTest.yaml  # to launch VM
+multipass shell ltTEST  # to enter VM's shell
+```
+
+And that's it! At the moment, Multipass isn't patient enough for an automated install,
+so after you enter the shell for the first time, you will still need to run the install script for LibreTime.
+
+```
+sudo ./libretime/install -fiap
+
+sudo service airtime-liquidsoap start
```

Comment:
These start commands are not needed, as they will do nothing until the user has gone through the web installer.